src/component/navBar.js

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import InputBase from '@mui/material/InputBase';
+import Button from '@mui/material/Button';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import SearchIcon from '@mui/icons-material/Search';
+import { useState, useEffect } from 'react';
+
+const Search = styled('div')(({ theme }) => ({
+  marginLeft: 'auto',
+  display: 'flex',
+  alignItems: 'center',
+  borderRadius: theme.shape.borderRadius + 20,
+  backgroundColor: 'rgba(229, 229, 229, 1)',
+  paddingLeft: theme.spacing(2),
+  paddingRight: theme.spacing(1),
+}));
+
+const StyledInputBase = styled(InputBase)(({ theme }) => ({
+  color: 'rgba(0, 0, 0, 0.87)',
+  '& .MuiInputBase-input': {
+    padding: theme.spacing(1),
+    transition: theme.transitions.create('width'),
+    width: '12ch',
+    '&:focus': {
+      width: '20ch',
+    },
+  },
+}));
+
+export default function SearchAppBar() {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [genre, setGenre] = useState('');
+  const [records, setRecords] = useState([]);
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleGenreChange = (event) => {
+    const selectedGenre = event.target.innerText;
+    console.log('Selected genre:', selectedGenre);
+    setGenre(selectedGenre);
+    handleClose();
+  };
+
+  useEffect(() => {
```

Comment:
It is a better practice to abstract the API requests into a hook.
We can do something similar to this but using fetch instead of Axios
https://medium.com/@nelson_examiner/building-custom-react-hooks-for-calling-apis-ab82a6b45ff